v2/brigadier/src/workers.ts

```diff
@@ -1,7 +1,20 @@
+/**
+ * Represents the Brigade worker within which an event handler runs.
+ */
 export interface Worker {
+  /** The address of the Brigade API server. */
   apiAddress: string
+  /** A token which can be used to authenticate to the API server. */
```

Comment:
Non-blocking-- it's up to you whether you think it's worth mentioning that this token _only_ allows this worker to create jobs for the current event _and nothing else._